source/developer_manual/development_tips/coding_style_guide.rst

```diff
@@ -0,0 +1,509 @@
+.. _coding_style_guide:
```

Comment:
is this now twice here?

Reply:
Yes kind of as the comment below stated.

Reply:
or above, but lets just have the confluence coding style guide, and drop the other.